[16.0][MIG] sale_invoice_blocking

dhx opened the pull request:
Based on the PR from @dsolanki-initos : #2469
(rebasing and resolving the README conflict)

damdam-s committed:
[ADD] new module sale_invoice_blocking

allow to block the invoicing of a sale order

oca-travis committed:
[UPD] Update sale_invoice_blocking.pot

[UPD] README.rst

[ADD] icon.png

Added translation using Weblate (Catalan)

Added translation using Weblate (Spanish)

Added translation using Weblate (Spanish)

Translated using Weblate (Catalan)

Currently translated at 100.0% (14 of 14 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_invoice_blocking
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_invoice_blocking/ca/

Translated using Weblate (Spanish)

Currently translated at 100.0% (14 of 14 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_invoice_blocking
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_invoice_blocking/es/

[UPD] README.rst

[UPD] README.rst

leemannd committed:
[MIG] sale_invoice_blocking: Migration to 15.0

Commit:
[UPD] Update sale_invoice_blocking.pot

OCA-git-bot committed:
[UPD] README.rst

dsolanki-initos committed:
[IMP] sale_invoice_blocking: pre-commit stuff

rousseldenis commented:
/ocabot migration sale_invoice_blocking

rousseldenis commented:
@dhx Could you rebase ?

dhx commented:
@rousseldenis rebased :)

dhx committed:
[MIG] sale_invoice_blocking: Migration to 16.0

OKT96 reviewed (approved):
LGTM

BhaveshHeliconia reviewed (approved):
Functional review LGTM!

dhx commented:
Hi @rousseldenis do you see anything preventing this from becoming merged?

rousseldenis commented on README.md:
```diff
@@ -48,6 +48,7 @@ addon | version | maintainers | summary
 [sale_force_invoiced](sale_force_invoiced/) | 16.0.2.1.2 |  | Allows to force the invoice status of the sales order to Invoiced
 [sale_force_invoiced_quantity](sale_force_invoiced_quantity/) | 16.0.1.0.0 |  | Add manual invoice quantity in sales order lines
 [sale_global_discount](sale_global_discount/) | 16.0.1.0.0 |  | Sale Global Discount
+[sale_invoice_blocking](sale_invoice_blocking/) | 16.0.1.0.0 |  | Allow you to block the creation of invoices from a sale order.
```
@dhx Don't update this as it will be updated automatically

rousseldenis commented on sale_invoice_blocking/__manifest__.py:
```diff
@@ -4,7 +4,7 @@
     "name": "Sale Invoice Blocking",
     "summary": "Allow you to block the creation of invoices from a sale order.",
-    "version": "15.0.1.0.0",
+    "version": "16.0.1.0.0",
```
You should put this into migration commit instead.

dhx commented:
@rousseldenis thanks for the feedback!
- Rebased
- Moved the version change in the manifest to the migration commit.
- Removed the change of the root README file.

rousseldenis reviewed (approved):
LGTM

rousseldenis commented:
/ocabot merge patch
